beacon/goclient/proposer.go

```diff
@@ -34,20 +34,71 @@ func (gc *goClient) GetBeaconBlock(slot phase0.Slot, committeeIndex phase0.Commi
 
 	switch beaconBlock.Version {
 	case spec.DataVersionPhase0:
+		if b := beaconBlock.Bellatrix.Body; b != nil && b.ExecutionPayload != nil {
+			gc.log.Info("got beacon block",
```

Comment:
is there a case where the body or payload are empty? shouldn't we log or do something in this case?

Reply:
I need to check this and I'm not sure if this log will remain